baserun/helpers.py

```diff
@@ -14,4 +19,28 @@ class BaserunType(Enum):
 class BaserunStepType(Enum):
     LOG = auto()
     AUTO_LLM = auto()
-    CUSTOM_LLM = auto()
\ No newline at end of file
+    CUSTOM_LLM = auto()
+
+
+def baserun_threaded_wrapper(
+    target: Callable, results: list, thread_args=None, thread_kwargs=None
```

Comment:
Is it possible for this target fn to be async? Seems generally weird to do that though

Reply:
If they want it to be async they would need to make the target `asyncio.run` with their callable target as an arg. Essentially, nothing changes with the wrapper